README.md

```diff
@@ -139,5 +144,35 @@ en la creación de la imagen. La imagen se encuentra almacenada en Dockerhub y p
 
 ---
 
+## Sistema de integración continua 
+Hoy en día existe una gran variedad de sistemas de integración continua que pueden usarse para nuestros proyectos, entre los que se pueden encontrar los siguientes:
+* Github Actions
+* Travis
+* Circle CI
+* Semaphore CI
+* AppVeyor
+* Jenkings
+
+Ahora la pregunta sería, ¿cuál puede ser el más recomendado para nuestro proyecto? 
+
+Atendiendo a la facilidad de uso y lo estándar que es el sistema, **Travis** es probablemente el más recomendado de usar, ya que tiene una gran integración con Github, permitiendo 
```

Comment:
Por favor, no pongáis frases gratuitas como "gran integración con GitHub". Lo que tenéis que exponer en cualquier selección de herramienta es hacer las tres fases que se explicaron en clase.

Reply:
Entendido :+1: